lib/matplotlib/figure.py

```diff
@@ -448,29 +448,35 @@ def autofmt_xdate(self, bottom=0.2, rotation=30, ha='right'):
         bottom subplot and turn them off on other subplots, as well as
         turn off xlabels.
 
-        *bottom*
+        Parameters
+        ----------
+
+        bottom : scalar
             The bottom of the subplots for :meth:`subplots_adjust`
 
-        *rotation*
+        rotation : angle in degrees
             The rotation of the xtick labels
 
-        *ha*
+        ha : string
             The horizontal alignment of the xticklabels
+
+        which : {None, 'major', 'minor', 'both'}
+            Selects which ticklabels to return
```

Comment:
Tom meant that you should change "Selects which ticklabels to return" to "Selects which ticklabels to rotate"